hw12/compare_script2.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#Compare numbers A & B & C.
+echo "Enter number A:";
+read a;
+
+echo "Enter number B:";
+read b;
+
+echo "Enter number C;";
+read c;
+
+if (( a > b )) && (( a > c )); then
+ echo "A is greater than B and C"
```

Comment:
Please compare in a three ways: 'A>B, B<C, A=C'

Reply:
at once?

Reply:
yes